dlog/commitment/src/commitment.rs

```diff
@@ -28,9 +28,7 @@ type Fr<G> = <G as AffineCurve>::ScalarField;
 type Fq<G> = <G as AffineCurve>::BaseField;
 
 #[derive(Clone, Debug)]
-#[cfg_attr(feature = "ocaml_types", derive(ocaml::IntoValue, ocaml::FromValue))]
-pub struct PolyComm<C>
```

Comment:
Is there any need to get rid of the derivation of the ocaml traits? It seems the CamlPolyComm type is identical so we could just use this type instead right?

Reply:
IIRC yeah because C must implement AffineCurve, which our wrapper does not

Reply:
it's not clear in this snippet, but on line 60 in the same file: 

```rust
impl<C: AffineCurve> PolyComm<C> {
```